Sources/PayPal/Models/PayPalResult.swift

```diff
@@ -0,0 +1,24 @@
+#if canImport(PayPalCheckout)
+@_implementationOnly import PayPalCheckout
+#endif
+
+import Foundation
+
+/// The result of a PayPal payment flow.
+public struct PayPalResult {
```

Comment:
We want to add `orderStatus` to `PayPalResult`:

```
    /// The current status of the transaction
    public let status: OrderStatus
```

Reply:
currently the PayPalCheckout SDK uses our internal GQL layer which doesn't have OrderStatus, but has this:
```
enum CartState {
CREATED
APPROVED
FAILED
SUBMITTED
}
```
Currently we're not fetching this in PayPalCheckout so it's not possible to get this for now.
But I believe our GQL approve query + `CartState` here are probably wrapper around Orders v2 + `OrderStatus`. I can investigate this and make this addition, but maybe it should be a separate ticket? :smile:

Reply:
oh just saw you removed `OrderStatus`. Good idea :smile: